stdlib/format.mli

```diff
@@ -1472,6 +1472,79 @@ val kasprintf : (string -> 'a) -> ('b, formatter, unit, 'a) format4 -> 'b
   @since 4.03
 *)
 
+(** Formatted Pretty-Printing with heterogeneous list as arguments.
+
+  The following functions behave the same as their non-'l' counter-parts, but
+  receive a heterogeneous list as arguments.
```

Comment:
Maybe `receive their arguments as a heterogeneous list` to better emphasis that those functions merely change the syntactic form of the list of arguments.

Reply:
I agree.